packages/dnb-eufemia/src/components/info-card/InfoCard.tsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react'
+import classnames from 'classnames'
+
+// Components
+import Button from '../button/Button'
+import { createSpacingClasses } from '../space/SpacingHelper'
+import { createSkeletonClass } from '../skeleton/SkeletonHelper'
+
+// Icons
+import {
+  lightbulb as LightbulbIcon,
+  close as CloseIcon,
+} from '../../icons'
+// Shared
+import Context from '../../shared/Context'
+import { ISpacingProps, SkeletonTypes } from '../../shared/interfaces'
+import { extendPropsWithContext } from '../../shared/component-helper'
+import IconPrimary, { IconPrimaryIcon } from '../icon-primary/IconPrimary'
+import Img, { ImgProps } from '../../elements/Img'
+import { Div, H3, P } from '../../elements'
+import { ButtonProps } from 'packages/dnb-eufemia/build/components/button'
+
+export interface InfoCardProps {
+  /**
+   * Aligns the content to center, rather than left
+   * Default: false
+   */
+  center?: boolean
+  /**
+   * Custom className on the component root
+   * Default: null
+   */
+  className?: string
+  /**
+   * Replace the default icon with custom icon.
+   * Default: Lightbulb (icon)
+   */
+  icon?: IconPrimaryIcon
+  /**
+   * Props applied to the `img` element if the component is used to display an image. Replace the 'icon'
+   * Default: null
+   */
+  imgProps?: ImgProps
+  /**
+   * Skeleton should be applied when loading content
+   * Default: null
+   */
+  skeleton?: SkeletonTypes
+  /**
+   * Specifies the path to the image
+   * Default: null
+   */
+  src?: string
+  /**
+   * Image src, will replace the 'icon' with the image
+   * Default: null
+   */
+  text: string
+  /**
+   * Component title
+   * Default: null
+   */
+  title?: string
+  /**
+   * Is called when the close button is clicked
+   * Default: null
+   */
+  onClose?: () => void
+  /**
+   * The text of the close button.
+   * Default: null
+   */
+  closeButtonText?: string
+  /**
+   * Is called when the accept button is clicked
+   * Default: null
+   */
+  onAccept?: () => void
```

Comment:
```suggestion
  onAccept?: React.MouseEventHandler<HTMLButtonElement>
```

I see we use the following, React.MouseEventHandler<HTMLButtonElement>, in https://github.com/dnbexperience/eufemia/blob/b5190df0586c03a970ca088116f133913469c682/packages/dnb-eufemia/src/components/breadcrumb/Breadcrumb.tsx#L56, and https://github.com/dnbexperience/eufemia/blob/59825b88d99148d5e26a8f638ad39507e4661bc3/packages/dnb-eufemia/src/components/breadcrumb/BreadcrumbItem.tsx#L41

Reply:
Done